tests/end2end/test_group.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2023 IBM Corp. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+End2end tests for groups.
+
+These tests do not change any existing groups, but create,
+modify and delete test groupus.
+"""
+
+from __future__ import absolute_import, print_function
+
+import warnings
+import pytest
+from requests.packages import urllib3
+
+import zhmcclient
+# pylint: disable=line-too-long,unused-import
+from zhmcclient.testutils import hmc_definition, hmc_session  # noqa: F401, E501
+# pylint: enable=line-too-long,unused-import
+
+from .utils import pick_test_resources, runtest_find_list, TEST_PREFIX, \
+    skip_warn
+
+urllib3.disable_warnings()
+
+# Properties in minimalistic PasswordRule objects (e.g. find_by_name())
+GROUP_MINIMAL_PROPS = ['object-uri', 'name']
+
+# Properties in PasswordRule objects returned by list() without full props
+GROUP_LIST_PROPS = ['object-uri', 'name']
+
+# Properties whose values can change between retrievals of PasswordRule objects
+GROUP_VOLATILE_PROPS = []
+
+
+def test_group_find_list(hmc_session):  # noqa: F811
+    # pylint: disable=redefined-outer-name
+    """
+    Test list(), find(), findall().
+    """
+    client = zhmcclient.Client(hmc_session)
+    hd = hmc_session.hmc_definition
+
+    api_version = client.query_api_version()
+    hmc_version = api_version['hmc-version']
+    hmc_version_info = tuple(map(int, hmc_version.split('.')))
+    if hmc_version_info < (2, 13, 0):
```

Comment:
could that go into a utility method?

Reply:
Good point. 
I moved these lines into a new function `skipif_no_group_support()`, similar to other such functions.